src/components/Pressable/PressableWithFeedback.js

```diff
@@ -12,28 +13,50 @@ const omittedProps = ['style', 'pressStyle', 'hoverStyle', 'focusStyle', 'wrappe
 const PressableWithFeedbackPropTypes = {
     ..._.omit(GenericPressablePropTypes.pressablePropTypes, omittedProps),
     pressDimmingValue: propTypes.number,
-    hoverDimmingValue: propTypes.number,
 };
 
 const PressableWithFeedbackDefaultProps = {
     ..._.omit(GenericPressablePropTypes.defaultProps, omittedProps),
     pressDimmingValue: variables.pressDimValue,
-    hoverDimmingValue: variables.hoverDimValue,
     wrapperStyle: [],
 };
 
 const PressableWithFeedback = forwardRef((props, ref) => {
     const propsWithoutStyling = _.omit(props, omittedProps);
+    const [disabled, setDisabled] = useState(props.disabled);
+
+    useEffect(() => {
+        setDisabled(props.disabled);
+    }, [props.disabled]);
+
     return (
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        <GenericPressable ref={ref} style={props.wrapperStyle} {...propsWithoutStyling}>
+        <GenericPressable
+            ref={ref}
+            style={props.wrapperStyle}
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...propsWithoutStyling}
+            disabled={disabled}
+            onPress={(e) => {
+                setDisabled(true);
+                const onPress = props.onPress(e);
+                InteractionManager.runAfterInteractions(() => {
+                    if (!(onPress instanceof Promise)) {
+                        setDisabled(props.disabled);
+                        return;
+                    }
+                    onPress.then(() => {
+                        setDisabled(props.disabled);
+                    });
+                });
+            }}
+        >
             {state => (
                 <OpacityView
-                    shouldDim={state.pressed || state.hovered}
-                    dimmingValue={state.pressed ? props.pressDimmingValue : props.hoverDimmingValue}
+                    shouldDim={!disabled && state.pressed}
+                    dimmingValue={props.pressDimmingValue}
```

Comment:
I wouldn't strip this from the functionality of distinguish dimming value between hover and press. As now they are unified on the `variables.js` level I think old logic can stay

Reply:
however @shawnborton mentioned that we do not want any dimming on hover, there is only a colour change when hovered, which is applied in `Button` styles

Reply:
In button case it's true, but Pressable component can be used in different places also, where it can come in handy IMO

Reply:
could you please give an example? or can we all discuss this in Slack & get a conclusion?